website/docs/components/accordion/partials/guidelines/guidelines.md

```diff
@@ -95,4 +140,48 @@ The `content` supports any custom content, local components, or Helios component
 
 ### Custom
 
-![Example of content in accordion item](/assets/components/accordion/content-custom.png =690x*)
\ No newline at end of file
+![Example of content in accordion item](/assets/components/accordion/content-custom.png =690x*)
+
+### Nesting Accordions
+
+!!! Info
+
+Nesting is only available in code. Figma does not support nesting an instance inside itself. You can detach the HDS component and turn it into a local component, then insert the local component within the HDS component. We strongly advise against this as it means your local Accordion component will not receive future component updates. [Link to Code example]
```

Comment:
[suggestion]

```suggestion
Nesting Accordions is only supported in code as Figma does not support nesting an instance inside itself. Work around this by detaching the HDS component and turn it into a local component, then insert the local component within the HDS component. However, we strongly advise against this as it means your local Accordion component will not receive future component updates. [Link to Code example]
```

Though, this brings up the question, can we "strongly advise" against this if we're giving them instructions for how to do it?

Reply:
That's a valid point, maybe leaning towards recommended then and add in "better in code" 

```suggestion
Nesting Accordions is only supported in code, as Figma does not support nesting an instance inside itself. While you can work around this by detaching the HDS component and turning it into a local component, then inserting the local component within the HDS component, we do not recommend this approach. Doing so means your local Accordion component will not receive future updates. It's best to handle nested Accordions in code to maintain update compatibility. [Link to Code example]
```